test/test_utils.py

```diff
@@ -130,20 +116,99 @@ def test_get_ip_without_internet(self, mock_get):
 class TestAssertConfigsEquivalent(unittest.TestCase):
     def test_same_configs(self):
         self.assertTrue(
-            assert_configs_are_equivalent_besides_ids(XRAY_CONFIG, XRAY_CONFIG)
+            assert_configs_are_equivalent_besides_ids(
+                copy.deepcopy(XRAY_CONFIG), copy.deepcopy(XRAY_CONFIG)
+            )
         )
 
     def test_equivalent_configs(self):
         self.assertTrue(
-            assert_configs_are_equivalent_besides_ids(XRAY_CONFIG, XRAY_CONFIG_DIFF_IDS)
+            assert_configs_are_equivalent_besides_ids(
```

Comment:
`test_equivalent_configs` and `test_same_configs` are identical right? If so we should delete one.

Reply:
Not quite, `test_equivalent_configs` is testing between `XRAY_CONFIG` and `XRAY_CONFIG_DIFF_IDS`. The configs have the same components, but different IDs